src/components/Filter/TagList.vue

```diff
@@ -59,11 +59,15 @@
 <script>
 import { isSingleCharacter } from 'docc-render/utils/input-helper';
 import handleScrollbar from 'docc-render/mixins/handleScrollbar';
+import keyboardNavigation from 'docc-render/mixins/keyboardNavigation';
 import Tag from './Tag.vue';
 
 export default {
   name: 'Tags',
-  mixins: [handleScrollbar],
+  mixins: [
+    handleScrollbar,
+    keyboardNavigation,
```

Comment:
We can remove the `focusedIndex` from this component, since its in the mixin.

Reply:
https://github.com/apple/swift-docc-render/pull/79/commits/1f9ba274e05fe857da47b80148eef7f8ac1d036a